src/legacy/ui/public/vis/editors/default/_agg_params.scss

```diff
@@ -10,3 +10,12 @@
   color: $textColor;
   background-color: $backgroundColor;
 }
+
+.visEditorAggParam--inline {
+  display: inline-block;
+  width: 50%;
```

Comment:
By making them fully 50%, the one with padding will actually not be the same size as the other. You can use a `calc` instead like:
```suggestion
  width: calc(50% - #{$euiSizeS / 2});
```

which will take into account the margin added to the last item.

Reply:
Thanks, done!

P.S. previously these two didn't have the same size, that's why I didn't take care of calculating them.
![image](https://user-images.githubusercontent.com/31325372/56645180-de6ed600-6685-11e9-84f0-4801aa53099b.png)